agent/pkg/routes/standalone_routes.go

```diff
@@ -0,0 +1,13 @@
+package routes
+
+import (
+	"github.com/gin-gonic/gin"
+	"mizuserver/pkg/controllers"
+)
+
+func StandaloneRoutes(ginApp *gin.Engine) {
+	routeGroup := ginApp.Group("/standalone")
+
+	routeGroup.POST("/updateConfig", controllers.UpdateConfig)
```

Comment:
maybe we should call it updateTapConfig?
next we will have many different configurations, users, workspace etc..